src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/gradle/GradleDependenciesParser.java

```diff
@@ -120,20 +104,29 @@ public DetectCodeLocation parseDependencies(final DetectProject detectProject, f
                 }
 
                 if (StringUtils.isBlank(line)) {
-                    processingConfiguration = false;
-                    configurationName = null;
-                    previousLine = null;
-                    nodeStack = new Stack<>();
-                    previousNode = null;
-                    treeLevel = 0;
+                    clearConfigurationState();
```

Comment:
Why are we now clearing the root project information? It doesn't seem like we were before. You could split clearConfigurationState() into two methods. One to clear the entire state and one to clear the things we want to clear here

Reply:
It just moved, it's as you describe below 👍